index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <p>Hello,world, How are you!</p>
+</body>
```

Comment:
Please, instead of putting your text in a _p_ tag, it would be better if you put it in an _h1_ tag, as it's an important heading.